site.yml

```diff
@@ -45,6 +45,14 @@
       tags: ['infra', 'rabbitmq']
   environment: "{{ env_vars|default({}) }}"
 
+- name: setup serverspec
+  hosts: all
+  roles:
+    - role: serverspec
+      tags: ['serverspec']
+      when: serverspec.enabled|bool
+  environment: "{{ env_vars|default({}) }}"
```

Comment:
That reasoning seems dangerous. It's an unexpressed role dependency, which means if we just run the `serverspec` tag, things will fail. But I don't see any calls to a rabbit handler in this PR. In fact, I only see one call to a handler here.